pkg/account/downloader/environments.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package downloader
+
+type (
+	Environments []environment
+
+	environment struct {
+		id              string
+		name            string
+		managementZones []managementZone
+	}
+
+	managementZone struct {
+		name     string
+		originID string
+	}
+)
+
+func (e environment) String() string {
+	return e.id
+}
+
+func (e Environments) asList() []string {
```

Comment:
this is just used in a debug message write?
maybe what you want is a String() method for Environments type?